src/components/Expander/examples/3.interactive-without-changing-labels.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import createClass from 'create-react-class';
-import { Expander } from '../../../index';
+import { Expander, ExpanderStateful } from '../../../index';
 
+//@todo child components are kinda awkward since they won't be directly on HoC versions.  Should they be their own export?
```

Comment:
note^^

Reply:
Again, in a vacuum I think the best approach would be to disconnect the child components from the root level component and just export them from the same file. Like `export { Expander, ExpanderLabel }` but it'd be a big lift for consumers to change their code unless we provided a codemod.

Reply:
another option might be to have `withReducer` somehow tack on the child components on the wrapped HoC.